doc/changelog/06-Ltac2-language/19589-ltac2-eval-flags.rst

```diff
@@ -0,0 +1,6 @@
+- **Fixed:**
+  resolution of :ref:`abbreviations <Abbreviations>` in Ltac2 eval tactic
+  delta-reduction flags :token:`ltac2_delta_reductions`
```

Comment:
I believe from the code that it's not just the reduction flags, it's anything that parses qualid to produce a globref

Reply:
Indeed, fixed